process/scToProtocol/stakingToPeer.go

```diff
@@ -178,7 +179,9 @@ func (stp *stakingToPeer) UpdateProtocol(body *block.Body, nonce uint64) error {
 		// no data under key -> peer can be deleted from trie
 		if len(data) == 0 {
 			err = stp.peerState.RemoveAccount(blsPubKey)
-			log.LogIfError(err, "staking to protocol RemoveAccount blsPubKey", blsPubKey)
+			if err != nil {
+				log.Debug("staking to protocol RemoveAccount", "error", err, "blsPubKey", hex.EncodeToString(blsPubKey))
```

Comment:
Please use stp.pubkeyConv.Encode instead of hex.EncodeToString(blsPubKey)

Reply:
this is bls key, not bech32.

Reply:
pubkeyConv is of type hexPkConverter. Can be left as it is